cram_common/cram_object_knowledge/src/environment.lisp

```diff
@@ -42,7 +42,17 @@
 (defmethod man-int:get-action-gripping-effort :heuristics 20 ((object-type (eql :container)))
   50)
 
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (defmethod man-int:get-action-gripper-opening :heuristics 20 ((object-type (eql :container)))
   0.10)
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defmethod man-int:get-container-opening-distance :heuristics 20 (container-designator)
```

Comment:
Should define one for ``:container-prismatic`` and one for ``:container-revolute``.
And the rest should be inferred using ``most-specific`` stuff.
E.g., you call ``(get-container-opening-distance :fridge)`` and get the stuff for ``:container-revolute``.